### PR TITLE
Remove Python 2 base image

### PR DESCRIPTION
**Pull request by allait**
All existing applications have been updated to use the Python 3
base image, so we can remove the steps to build and push Python 2
images to Docker Hub.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,30 +23,12 @@ define push-image
...
-	$(eval export BASE_IMAGE_SUFFIX=-py2)
 	$(call build-image)
 
-.PHONY: push-all
-push-all: push-py2 push-py3
-
-.PHONY: push-py3
-push-py3:
+.PHONY: push
+push:
```
Comment (Wynndow) [inline Makefile]: Same, we could lose the function.

File: Makefile [hunk 2]
```diff
@@ -1,9 +1,8 @@
 VERSION := $(shell cat VERSION)
-PY2VERSION := $(shell cat PY2VERSION)
 
 define build-image
```
Comment (Wynndow) [inline Makefile]: Can we move this into the build step? I only defined it so it could be paramaterised for py2 and 3 builds.

Comment (allait) [inline Makefile]: Yep, makes sense. Inlined both functions.

File: Makefile [hunk 3]
```diff
@@ -1,19 +1,25 @@
-VERSION := $(shell cat VERSION)
-PY2VERSION := $(shell cat PY2VERSION)
+BUILD_VERSION := $(shell cat VERSION)
 
 define build-image
```
Comment (Wynndow) [inline Makefile]: These definitions could be dropped?

File: Makefile [hunk 4]
```diff
@@ -1,19 +1,25 @@
...
 
 	docker build -t digitalmarketplace/base-frontend -f frontend.docker .
 	docker tag digitalmarketplace/base-frontend digitalmarketplace/base-frontend:${BUILD_VERSION}
-endef
 
-define push-image
+.PHONY: push
+push:
 	docker push digitalmarketplace/base:${BUILD_VERSION}
 	if [ "$${BUILD_VERSION:0:1}" -ne "1" ]; then docker push digitalmarketplace/base:latest; fi
```
Comment (Wynndow) [inline Makefile]: Can we drop these checks as we're only going to building/pushing > version 2.x.x?

File: Makefile (hunk 3 above)
Comment (allait) [inline Makefile]: Yup :man_facepalming:

File: Makefile (hunk 4 above)
Comment (allait) [inline Makefile]: Yeah, good call. Cleaned them up.

**Pull request merged**